src/components/Form.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {
+  useState, useRef, useEffect, RefObject,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { createTodo } from '../api/todos';
+
+type Props = {
+  USER_ID: number;
+  addNewTodo: (item: Todo) => void;
+  showErrorNotification: (value: string) => void;
+  setTempTodo:(item: Todo | null) => void;
+};
+
+export const Form: React.FC<Props> = ({
+  USER_ID,
+  addNewTodo,
+  showErrorNotification,
+  setTempTodo,
+}) => {
+  const formRef: RefObject<HTMLInputElement> = useRef(null);
```

Comment:
You could pass a generic to the `useRef` as well
```suggestion
  const formRef = useRef<HTMLInputElement>(null);
```